openshift/WFLY-15421_OIDC_for_WildFly_s2i_v2.adoc

```diff
@@ -0,0 +1,306 @@
+= OIDC support for WildFly S2I V2 image
+:author:           Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+For WildFly s2i V2 we are removing the keycloack OIDC and SAML client adapters present in the s2i image.
+The only supported integration with keycloak would be based on the `elytron-oidc-client` subsystem and 
```

Comment:
This a bit imprecise as the SAML part is out of scope of this RFE and is handled within the scope of WFLY-16306.

Reply:
Thank-you, I removed the SAML mention.